src/Parser/Druid/Restoration/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,166 @@
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.CENARION_WARD_TALENT,
+            when: combatant.hasTalent(SPELLS.CENARION_WARD_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.FLOURISH_TALENT,
+            when: combatant.hasTalent(SPELLS.FLOURISH_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.ESSENCE_OF_GHANIR,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.TRANQUILITY_CAST,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.INNERVATE,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Manage your mana',
+      description: <Wrapper>Casting on targets who don't need healing or recklessly using inefficienct heals can result in running out of mana well before an encounter ends. Adapt your spell use to the situation, and as a rule of thumb try and keep your current mana percentage just above the bosses health percentage.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: <Wrapper>Mana saved during <SpellLink id={SPELLS.INNERVATE.id} icon /></Wrapper>,
+            check: () => this.innervate.averageManaSavedSuggestionThresholds,
+            tooltip: `All spells cost no mana during Innervate, so take care to chain cast for its duration. Typically this means casting a Wild Growth, refreshing Efflorescence, and spamming Rejuvenation. It's also fine to Regrowth a target that is in immediate danger of dying.`,
+          }),
+          new Requirement({
+            name: <Wrapper>Use your <SpellLink id={SPELLS.CLEARCASTING_BUFF.id} icon /> procs</Wrapper>,
+            check: () => this.clearcasting.clearcastingUtilSuggestionThresholds,
+            tooltip: `This is the percentage of Clearcasting procs that you used. Regrowth is normally very expensive, but it's completely free with a Clearcasting proc. Use your procs in a timely fashion for a lot of free healing.`,
+          }),
+          new Requirement({
+            name: <Wrapper>Don't overuse <SpellLink id={SPELLS.REGROWTH.id} icon /></Wrapper>,
+            check: () => this.clearcasting.nonCCRegrowthsSuggestionThresholds,
+            tooltip: `This is the number of no-clearcasting Regrowths you cast per minute. Regrowth is very mana inefficient, and should only be used in emergency situations (and when you've already expended Swiftmend). Usually, you should rely on other healers in your raid to triage.`,
+          }),
```

Comment:
Could also include "Use all of your mana effectively" i.e. making sure the player has used up most of their mana at the end of the encounter.

Reply:
I would probably move the "Used a second potion" here as it's directly tied with mana management.

Reply:
Oh yeah, forgot about the mana use rate one, I'll put that in. The second potion thing is tied to a generic rule, I don't think it's worth trying to mess with that.